pkg/server/plugin/datastore/datastore.go

```diff
@@ -93,30 +91,28 @@ type BySelectors struct {
 	Match     MatchBehavior
 }
 
```

Comment:
Some sections of this file were rearranged for better readability after most request/response structures were removed. Apologies for the resulting jumbled diff in this PR.

Reply:
No worries!